### PR TITLE
Parser order

### LINKED ISSUE #100: Add order and stop_processing to parsers table
Add an order (integer) and stop_processing_after_executing (true/false) fields to the parsers table and edit the chat parser and react parser business rules to abide by these fields:
parsers execute in order from lower to higher then blank, and if stop_processing is true, break the loop.

### PR DESCRIPTION
**Pull request by DavidArthurCole**
- Add Order:"order" [Integer] and Stop Processing:"stop_processing" [True/False] fields to the Parsers [x_snc_slackerbot_parser] Table.
- Update "Parse Chat" Business Rule to run parsers in decreasing order
- Update "Parse Chat" Business Rule to stop processing if a parser triggers with Stop Processing enabled
- Update CONTRIBUTING.MD with documentation on the new fields and associated optional headers

close #100

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,7 +54,9 @@ Eg. Run a script when a user says ____
 - Line 2 must always be `activation_example:` followed by a short description of how this parser would be activate
 - Line 3 must always be `regex:` followed by a regex expression that validates if the parser should run. Do not include the opening and closing `/`
 - Line 4 must always be `flags:` followed by the regex flags needed for line 3. If no flags are needed then leave the line as `flags:`
-- Line 5 must always be `*/`
+- *Optional*: `order:` that the parser should run (higher orders run first, default value is `100`).
```
Comment (earlduque) [inline CONTRIBUTING.md:57]: This is opposite of how order behavior usually works in ServiceNow. We really should do the opposite

File: Parsers/Tell a Random Joke.js [hunk 2]
```diff
@@ -2544,4 +2544,4 @@ while (!selectedJoke){
 var sendIt = new x_snc_slackerbot.Slacker().send_chat(current, selectedJoke.joke, false);
 
 //Send the punchline
-var sendItTwoElectricBoogallo = new x_snc_slackerbot.Slacker().send_chat(JSON.parse(sendIt.getBody()), selectedJoke.punchline, !sendIt.haveError());
\ No newline at end of file
+var sendItTwoElectricBoogallo = new x_snc_slackerbot.Slacker().send_chat(JSON.parse(sendIt.getBody()), selectedJoke.punchline, !sendIt.haveError());
```
Comment (earlduque) [inline Parsers/Tell a Random Joke.js:2547]: no actual changes? why is this being included in the PR?

File: b02cf9e61b861d10d806dc23b24bcb3f/update/sys_script_b64906e21bc61d10d806dc23b24bcbd9.xml [hunk 3]
```diff
@@ -31,14 +31,20 @@
 	
 	var parser = new GlideRecord('x_snc_slackerbot_parser');
 	parser.addQuery('active', 'true');
+	//Loop in decreasing order
+	parser.orderByDesc('order');
```
Comment (earlduque) [inline b02cf9e61b861d10d806dc23b24bcb3f/update/sys_script_b64906e21bc61d10d806dc23b24bcbd9.xml:35]: This is opposite of how order behavior usually works in ServiceNow. We really should do the opposite

**Review by earlduque — CHANGES_REQUESTED** [review]

File: Parsers/Tell a Random Joke.js (hunk 2 above)
Comment (DavidArthurCole) [reply Parsers/Tell a Random Joke.js:2547]: This one caused a merge conflict for some reason - just left it the same way it was in `main`

**Review by earlduque — APPROVED** [review]

**Pull request merged**